hack/release-automater/release_overview.py

```
@@ -0,0 +1,369 @@
+    results = []
+    images_data = get_grade_data()
+    grade_data = extract_grade_data(images_data.get('data', images_data))
+
+    for version in product["versions"]:
+        current_phase = get_current_phase(version, today)
+        eol_date = get_eol_date(version)
+        wmco_z_stream_version = "unknown"
+        latest_grade = "unknown"
+        try:
+            minor_version = version["name"].split('.')[1]
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)